src/jit/codegenlinear.h

```diff
@@ -146,6 +146,10 @@ void genSetRegToIcon(regNumber reg, ssize_t val, var_types type = TYP_INT, insFl
 
 void genCodeForShift(GenTreePtr tree);
 
+#if defined(_TARGET_X86_) && !defined(LEGACY_BACKEND)
+void genCodeForShiftLong(GenTreePtr tree);
+#endif
```

Comment:
(Minor) codegenLinear.h is a header file that is included only in RyuJIT backend.   For that reason we don't need !defined(LEGACY_BACKEND) in #ifdef above.